### PR TITLE
completed ticket number 3

### PR DESCRIPTION
**Pull request by DanielCouperthwaite**
Hi, I think I have completed all of task number 3 and am submitting a pull request.

### CODE REVIEW

File: controllers/controllers.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const {fetchTopics} = require('../models/models')
+
+exports.getTopics = (req, res) => {
+    console.log('in controller')
+    fetchTopics().then((topics) => {    
+        res.status(200).send({topics})
+    })
```
Comment (johnhutcheon) [inline controllers/controllers.js:7]: Can we just stick a catch block on here please?

File: app.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+const {getTopics} = require('./controllers/controllers')
+
+app.get('/api/topics', getTopics)
+
+console.log('in app')
+
+app.all('*', (req, res) => {
+    res.status(404).send({ msg: 'Oh no! Please enter a valid url' })
+});
+
```
Comment (johnhutcheon) [inline app.js:12]: Could we just add an app.use here to deal with any 500 errors?

**Review by johnhutcheon — CHANGES_REQUESTED** [review]
Helloooo! Really nice start! Could you just see my comments and make the changes please? Thanks! 🙂

File: controllers/controllers.js (hunk 1 above)
Comment (DanielCouperthwaite) [reply controllers/controllers.js:7]: Hi again John, I have submitted the corrections by pushing them to this branch. I don't think I need to make a new pull request right? I happily will if I need to!

**Pull request merged**